cmd/kubeadm/app/cmd/phases/certs_test.go

```diff
@@ -32,7 +32,7 @@ import (
 )
 
 // phaseTestK8sVersion is a fake kubernetes version to use when testing
-const phaseTestK8sVersion = "v1.10.0"
+const phaseTestK8sVersion = "v1.12.0"
```

Comment:
Why here jumps to 1.12? Typo?

Reply:
yeah, the tests still pas with it, but probably should either bump to .11 or leave it .10.